vipps-login-api-faq.md

```diff
@@ -40,23 +40,35 @@ the page which the user is redirected to after finishing a login. See:
 [API endpoints](vipps-login-api.md#api-endpoints)
 as well as requirements for the URIs below.
 
-You will find the "Setup Vipps Login" option in the same place as you find your
-`client_id` and `client_secret` (click "setup login" to get started):
+You will find the "Setup Vipps Login" option under the "Utvikler" menu,
+in the same place as you find your `client_id` and `client_secret`.
+Click "Setup login" for the sale unit you want to set up Vipps Login for.
+
+The "Setup login" button is on the right side of the sale unit overview:
 
 ![You will find the Setup Vipps Login option in the same place as you find your `client_id` and `client_secret`](images/portal_setup_login.png)
 
-First you activate Vipps Login:
+First you activate Vipps Login by clicking the "Activate Vipps Login" button:
+
 ![First you activate Vipps Login](images/portal_setup.jpeg)
 
 Then you can add the redirect URIs you need:
+1. Enter the URI. It must be _exactly_ the same URI that you use when making
+   API requests. Remember if t he actual URI specified with `redirect_uri` in
+   the API requests will have a trailing slash `/` or not.
+2. Click "Add URI" for the one or more URIs you are adding.
+3. Click "Save" to save all the URIs you have added.   
+
 ![Then you can add the redirect URIs you need](images/portal_direct_uris.jpeg)
 
 ## What are the requirements for redirect URIs?
 
-We validate redirect URIs against a whitelist of pre-approved URIs. The URIs
-must be registered by the merchant on [portal.vipps.no](https://portal.vipps.no).
+We validate redirect URIs used in API request against a whitelist of
```

Comment:
again, this is an `/auth`request where the end user is redirected, not an API request